ouroboros-consensus-cardano/test/cardano-test/Test/ThreadNet/Cardano.hs

```diff
@@ -478,31 +478,14 @@ mkProtocolCardanoAndHardForkTxs
                                           -- window so that the forks induced by
                                           -- the network partition are as deep
                                           -- as possible.
+        (SL.ProtVer conwayMajorVersion  0) -- FIXME: we need to justify why this is correct (if it's indeed correct)
```

Comment:
FTR, discussed with @amesgen. 

Yes, this makes sense, because previously I had set the (now single) protocol version to 0, and as soon as we forked out to Shelley, having protocol version 0 meant we did not support versions beyond Byron (which had version 0).

FTR, previous error was:

```text
 ,*** Failed! Exception: 'FailureException (ExceptionInLinkedThread "ThreadId [9]" (MiniProtocolFatalException {mpfeType = ChainSyncClientException, mpfeExn = HeaderError (At (Block {blockPointSlot = SlotNo 100, blockPointHash = cfcbf93090efa7a026e8cc8c59f3a014966172721a3807883cd42ac7642e68a5})) (HeaderEnvelopeError (OtherHeaderEnvelopeError (HardForkEnvelopeErrFromEra S (Z (WrapEnvelopeErr {unwrapEnvelopeErr = ObsoleteNodeCHAIN (Version 1) (Version 0)}))))) (Tip (SlotNo 99) 5732d7e7d15da554edd1ab57edeb24e4eb6283a270bc4d6c9f3130912d401df4 (BlockNo 92)) (Tip (SlotNo 99) 5732d7e7d15da554edd1ab57edeb24e4eb6283a270bc4d6c9f3130912d401df4 (BlockNo 92)), mpfeClient = CoreNodeId 1, mpfeServer = CoreNodeId 3}))' (after 1 test):
```